### PR TITLE
Airbyte Cron: renable schedule to file cleaner

### PR DESCRIPTION
**Pull request by marcosmarxm**
## What
https://github.com/airbytehq/airbyte/pull/17947 removed the Schedule annotation and the file cleaner is not schedule anymore.

## How
Renable the schedule for file cleaner.

## Recommended reading order
1. `x.java`
2. `y.python`

## 🚨 User Impact 🚨
Are there any breaking changes? What is the end result perceived by the user? If yes, please merge this PR with the 🚨🚨 emoji so changelog authors can further highlight this if needed.

## Pre-merge Checklist
Expand the relevant checklist and delete the others.

<details><summary><strong>New Connector</strong></summary>

### Community member or Airbyter

- [ ] **Community member?** Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- [ ] Secrets in the connector's spec are annotated with `airbyte_secret`
- [ ] Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [ ] Code reviews completed
- [ ] Documentation updated
    - [ ] Connector's `README.md`
    - [ ] Connector's `bootstrap.md`. See [description and examples](https://docs.google.com/document/d/1ypdgmwmEHWv-TrO4_YOQ7pAJGVrMp5BOkEVh831N260/edit?usp=sharing)
    - [ ] `docs/integrations/<source or destination>/<name>.md` including changelog. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
    - [ ] `docs/integrations/README.md`
    - [ ] `airbyte-integrations/builds.md`
- [ ] PR name follows [PR naming conventions](https://docs.airbyte.com/contributing-to-airbyte/issues-and-pull-requests)

### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items.

- [ ] Create a non-forked branch based on this PR and test the below items on it
- [ ] Build is successful
- [ ] If new credentials are required for use in CI, add them to GSM. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci).
- [ ] [`/test connector=connectors/<name>` command](https://docs.airbyte.io/connector-development#updating-an-existing-connector) is passing
- [ ] New Connector version released on Dockerhub by running the `/publish` command described [here](https://docs.airbyte.io/connector-development#updating-an-existing-connector)
- [ ] After the connector is published, connector added to connector index as described [here](https://docs.airbyte.io/connector-development#publishing-a-connector)
- [ ] Seed specs have been re-generated by building the platform and committing the changes to the seed spec files, as described [here](https://docs.airbyte.io/connector-development#publishing-a-connector)

</details>

<details><summary><strong>Updating a connector</strong></summary>

### Community member or Airbyter

- [ ] Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- [ ] Secrets in the connector's spec are annotated with `airbyte_secret`
- [ ] Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [ ] Code reviews completed
- [ ] Documentation updated
    - [ ] Connector's `README.md`
    - [ ] Connector's `bootstrap.md`. See [description and examples](https://docs.google.com/document/d/1ypdgmwmEHWv-TrO4_YOQ7pAJGVrMp5BOkEVh831N260/edit?usp=sharing)
    - [ ] Changelog updated in `docs/integrations/<source or destination>/<name>.md` including changelog. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
- [ ] PR name follows [PR naming conventions](https://docs.airbyte.com/contributing-to-airbyte/issues-and-pull-requests)

### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items.

- [ ] Create a non-forked branch based on this PR and test the below items on it
- [ ] Build is successful
- [ ] If new credentials are required for use in CI, add them to GSM. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci).
- [ ] [`/test connector=connectors/<name>` command](https://docs.airbyte.io/connector-development#updating-an-existing-connector) is passing
- [ ] New Connector version released on Dockerhub and connector version bumped by running the `/publish` command described [here](https://docs.airbyte.io/connector-development#updating-an-existing-connector)

</details>

<details><summary><strong>Connector Generator</strong></summary>

- [ ] Issue acceptance criteria met
- [ ] PR name follows [PR naming conventions](https://docs.airbyte.com/contributing-to-airbyte/issues-and-pull-requests)
- [ ] If adding a new generator, add it to the [list of scaffold modules being tested](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connector-templates/generator/build.gradle#L41)
- [ ] The generator test modules (all connectors with `-scaffold` in their name) have been updated with the latest scaffold by running `./gradlew :airbyte-integrations:connector-templates:generator:testScaffoldTemplates` then checking in your changes
- [ ] Documentation which references the generator is updated as needed

</details>

## Tests

<details><summary><strong>Unit</strong></summary>

*Put your unit tests output here.*

</details>

<details><summary><strong>Integration</strong></summary>

*Put your integration tests output here.*

</details>

<details><summary><strong>Acceptance</strong></summary>

*Put your acceptance tests output here.*

</details>

### CODE REVIEW

**Review by gosusnp — CHANGES_REQUESTED** [review]
This makes sense to me, @jdpgrailsdev,

File: airbyte-cron/src/main/java/io/airbyte/cron/selfhealing/WorkspaceCleaner.java [hunk 1]
```diff
@@ -52,6 +52,7 @@ public class WorkspaceCleaner {
    * NOTE: this is currently only intended to work for docker
    */
   @Trace(operationName = SCHEDULED_TRACE_OPERATION_NAME)
+  @Scheduled(fixedRate = "1d")
```
[the remark this answers is not in the document]
Comment (marcosmarxm) [reply airbyte-cron/src/main/java/io/airbyte/cron/selfhealing/WorkspaceCleaner.java:55]: This is in the class level. Should I move it to function level?

https://github.com/airbytehq/airbyte/blob/44a85e75f7d96f7f71ee6c5ef92b3d4a38117f5c/airbyte-cron/src/main/java/io/airbyte/cron/selfhealing/WorkspaceCleaner.java#L29-L35

[the remark this answers is not in the document]
Comment (jdpgrailsdev) [reply airbyte-cron/src/main/java/io/airbyte/cron/selfhealing/WorkspaceCleaner.java:56]: That should be fine.  The singleton will not be created (and therefore the scheduled task will not run) if the environment is Kubernetes.

**Comment by marcosmarxm** [comment]
After building the dev version:
```
2022-12-09 19:03:39 INFO i.a.c.s.WorkspaceCleaner(<init>):39 - Creating workspace cleaner
2022-12-09 19:03:39 INFO i.a.c.EnvConfigs(getEnvOrDefault):1143 - Using default value for environment variable TEMPORAL_HISTORY_RETENTION_IN_DAYS: '30'
2022-12-09 19:03:39 INFO i.a.c.s.WorkspaceCleaner(deleteOldFiles):59 - Deleting files older than 30 days (2022-11-09T19:03:39.812+0000)
2022-12-09 19:03:39 INFO i.a.c.t.TemporalUtils(getTemporalClientWhenConnected):224 - Waiting for temporal server...
2022-12-09 19:03:39 WARN i.a.c.t.TemporalUtils(getTemporalClientWhenConnected):235 - Waiting for namespace default to be initialized in temporal...
2022-12-09 19:03:42 INFO i.t.s.WorkflowServiceStubsImpl(<init>):86 - Created WorkflowServiceStubs for channel: ManagedChannelOrphanWrapper{delegate=ManagedChannelImpl{logId=1, target=airbyte-temporal:7233}}
2022-12-09 19:03:43 INFO i.a.c.s.WorkspaceCleaner(deleteOldFiles):76 - deleted 585 files
```
 
It is running again

File: airbyte-cron/src/main/java/io/airbyte/cron/selfhealing/WorkspaceCleaner.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (gosusnp) [reply airbyte-cron/src/main/java/io/airbyte/cron/selfhealing/WorkspaceCleaner.java:56]: Oh, didn't see it up there.

**Review by gosusnp — APPROVED** [review]

**Pull request merged**